SignalMessaging/ViewControllers/OWSNavigationController.m

```diff
@@ -150,7 +150,9 @@ - (void)navBarCallLayoutDidChangeWithNavbar:(OWSNavigationBar *)navbar
 
 - (UIStatusBarStyle)preferredStatusBarStyle
 {
-    if (OWSWindowManager.sharedManager.hasCall) {
+    if (!CurrentAppContext().isMainApp) {
+        return super.preferredStatusBarStyle;
+    } else if (OWSWindowManager.sharedManager.hasCall) {
```

Comment:
A bunch of fixes that I broke out for v2.32.0 into https://github.com/signalapp/Signal-iOS/pull/4026 .